sky/serve/serve_utils.py

```diff
@@ -611,6 +616,116 @@ def _get_replica_status() -> serve_state.ReplicaStatus:
     return ''
 
 
+def extract_replica_id_from_launch_log_file_name(file_name: str) -> int:
+    pattern = r'.*replica_(\d+)_launch.log'
```

Comment:
add a constant for this? ideally add a constant for function `generate_replica_launch_log_file_name` as well and put them together